### PR TITLE
Move updateCheckInterval to config #23

### PR DESCRIPTION
**Pull request by codisteinborn**
## Description
Moved 'notifier' as 'update_check_interval' to the config file to match the implementation of defaultpassword.

## Related Issue

#23 

## Motivation and Context

This helps standardize the code style of this project.

## How Has This Been Tested?

Tested by running dav-cli in console

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/lib/updateNotifier.js [hunk 1]
```diff
@@ -1,8 +1,3 @@
-const updateNotifier = require('update-notifier');
-const pkg = require('../../package.json');
+const config = require('../config');
 
-const notifier = updateNotifier({
-  pkg,
-  updateCheckInterval: 86400000, // 1 day
-});
-notifier.notify();
+const notifier = config('update_check_interval');
```
Comment (cg-cnu) [inline src/lib/updateNotifier.js]: Hey @codisteinborn You have completely replaced the notifier object.
You are supposed to change only `updateCheckInter : config('update_check_interval');` Please fix it and update your pr

**Review by cg-cnu — CHANGES_REQUESTED** [review]
Hey @codisteinborn Thanks for your interest in fixing this issue. 
You have completely removed the `updateNotifier` and `pkg`. The idea of this issue is to move the magic number `86400000` into the config file. Let me know if things are not clear, I will try to help 🙂

**Comment by codisteinborn** [comment]
I see what you mean. Updating now!

**Comment by cg-cnu** [comment]
Hey @codisteinborn there is a merge conflict. Can you resolve it and update the pr so that we can merge it. Thanks! 🙂

**Comment by cg-cnu** [comment]
Hey @codisteinborn Are you still interested in fixing this PR or please let us know, else will assign it to some one else! Thanks! 🙂

**Comment by codisteinborn** [comment]
Just resolved the merge conflict!

**Pull request merged**